release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1636,6 +1636,10 @@ With this release, availability sets are no longer modified after creation, allo
 [id="ocp-release-note-management-console-bug-fixes_{context}"]
 ==== Management Console
 
+* Previously, the Cluster Settings page would not properly render during a cluster update if the cluster's ClusterVersion lacked a `Completed` update. With this update, the Cluster Setting page properly renders even if the ClusterVersion lacks a `Completed` update. (link:https://issues.redhat.com/browse/OCPBUGS-49839[OCPBUGS-49839])
```

Comment:
Hi @tedaveryredhat . Thanks for raising this PR. Are you still adding bugs for management console to this PR? I see more bugs that need release notes, but maybe they were only wordsmithed today. Would you be OK to add them. See the [filter](https://issues.redhat.com/secure/Dashboard.jspa?selectPageId=12347491#SIGwKWmOqDAaIiGh1DSCAiYg6oaN4RY8HFpgaAw2CBElyBoBgDBga6GAtG0HT+ogCErGsyGEN+BWVIilR7PMcS+uOQx2tg2Aop6VhAA) here.